src/libserver/url.c

```diff
@@ -230,7 +230,7 @@ struct rspamd_url_flag_name {
 		{"schema_encoded", RSPAMD_URL_FLAG_SCHEMAENCODED, -1},
 		{"path_encoded", RSPAMD_URL_FLAG_PATHENCODED, -1},
 		{"query_encoded", RSPAMD_URL_FLAG_QUERYENCODED, -1},
-		{"missing_slahes", RSPAMD_URL_FLAG_MISSINGSLASHES, -1},
+		{"missing_slashes", RSPAMD_URL_FLAG_MISSINGSLASHES, -1},
```

Comment:
api?

Reply:
I think this flag is mostly unused, but I'd ask @citrin to confirm.

Reply:
Let's assume that I have received a telepathical approval from him :)